### ISSUE TITLE
Player used Shock spikes procs too often

### ISSUE THREAD

**Issue by Steam-arch** [EdenServer/community#4959]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Seems to have a extremely high proc rate on eden (tested on EP mobs, seemed to be around 60-80%, might be a similar issue as with ice spikes where the paralyze potency it procs is essentially uncapped. According to ancient forum guidance, the proc rate should be around 2-5%

Forum marks from gungnir & rdm shock spikes
![image](https://user-images.githubusercontent.com/69089249/159044096-a8488da4-c4b7-49fe-910c-68a8e547478c.png)
![image](https://user-images.githubusercontent.com/69089249/159044259-3dbbf21e-469d-4243-9273-548c9e1878b9.png)
![image](https://user-images.githubusercontent.com/69089249/159044457-62055d7c-da2e-44cf-856b-7327e793f0d2.png)
![image](https://user-images.githubusercontent.com/69089249/159044481-81550e2d-0cbb-41b1-a1bb-1a98a590b1e9.png)
![image](https://user-images.githubusercontent.com/69089249/159044900-31b2ffc2-0f64-4df1-b11c-96565cf0b691.png)
![image](https://user-images.githubusercontent.com/69089249/159048221-a24e64d1-b092-425a-b6ec-c4e5c87ee75d.png)
2013 thread
https://forum.square-enix.com/ffxi/threads/31682

![image](https://user-images.githubusercontent.com/69089249/159048728-d2b70a95-86f8-4460-967c-c57d7954d976.png)
2009 thread but not much in it.

![image](https://user-images.githubusercontent.com/69089249/159049101-1749af8d-637e-4b55-a60a-9e6690e7e95f.png)
2007 (?) mention
https://ffxiclopedia.fandom.com/wiki/Talk:Duelist%27s_Tights

Didnt find if it was changed or if it always just had an abysmal proc rate, but leaning towards the second one.

**Comment by tacbox3** [comment]
I'm pretty sure proc rate scales with enhancing skill vs. mob level and innate resistances? Also the damage calc and proc rates need to be treated as separate entities. I can confirm that gungnir shock spikes are complete garbage on Eden in terms of proc rate, I haven't had a stun once. DRG has no enhancing skill, so this is expected behavior. I'm not seeing anything in these threads saying the proc rate between gungnir and spell version is the same in era... Wouldn't expect that to be true. Can you verify that you are getting a very high proc rate with the spell version on higher level mobs? Which would be a more accurate comparison to the DRG aftermath

**Comment by Steam-arch** [comment]
Alright gotcha, they might be coded to have seperate properties then.
Atleast this: ![image](https://user-images.githubusercontent.com/69089249/159064766-ce87939a-b73b-48cc-b920-4e62e713e422.png)
Suggests that both are floored to a static % altho its a 2013 post and theres also the false information possibility.

I can check it on higher level monsters abit later aswell.

**Comment by Steam-arch** [comment]
![image](https://user-images.githubusercontent.com/69089249/159066300-d00c0449-f2a7-4fce-8e04-026b4b5d95c3.png)
Altho atleast this suggests that enhancing does affect gungnirs shock spikes proc rate. The post above this however implies rdm shock spikes proc rate doesnt scale with enhancing. hmm

**Comment by tacbox3** [comment]
Yeah my understanding for the aftermath was always that I could sub RDM or something and it would proc a bit more due to the higher skill. I haven't tested this direction Eden but am working towards being able to.

I think you'll see a much lower proc rate with the BLM spell on IT mobs. The 2-3% thing I suspect is in that context. If the proc rate is easily floored by skill or level differentials, then you're right that the aftermath and spell will look very similar on higher level mobs.

**Comment by Steam-arch** [comment]
![image](https://user-images.githubusercontent.com/69089249/159081631-53241ac6-d36b-4cce-b96c-ca2e121bc26d.png)
Atleast with 94 enhancing (blu75/blm37) against a lvl 65 WHM gob, stun rate is still around 50-80% on eden
